src/src/components/navigation/NavigationMenuItem.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <li 
+  :id="id+'_'+name"
+  class="navigation-link"
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 2 spaces.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
    class="navigation-link"
```